src/utils.js

```diff
@@ -22,4 +22,38 @@ export const getWikipediaAttrFromUrl = url => {
 	getDir = lang => {
 		const rtl = [ 'ar', 'arc', 'arz', 'dv', 'fa', 'ha', 'he', 'khw', 'ks', 'ku', 'ps', 'ur', 'yi', 'pnb', 'ckb', 'mzn', 'glk', 'ug', 'sd', 'azb', 'lrc' ]
 		return rtl.indexOf( lang ) === -1 ? 'ltr' : 'rtl'
+	},
+
+	defaultParams = {
+		format: 'json',
+		formatversion: 2,
+		origin: '*'
+	},
+
+	buildMwApiUrl = ( lang, params ) => {
+		// TODO - spread or Object.assign?
```

Comment:
Spread might have [browser compatibility](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax#Browser_compatibility) issues, the linter complains about `Object.assign()`